### PR TITLE
WPCS Fixes for BuddyPress Followers

### PR DESCRIPTION
**Pull request by renatonascalves**
I have a local, modified, version of BuddyPress Followers. While updating my version with the recent changes from BF, I noticed several WPCS bugs.

This PR tries to fix most of them.

### CODE REVIEW

**Comment by r-a-y** [comment]
Looks good! But can you change all instances of `buddypress()` to `$GLOBALS['bp']` instead?  BP Follow still supports v1.5 (for better or worse!).

Thanks @renatonascalves!

**Comment by renatonascalves** [comment]
@r-a-y About the global change, not a problem! Makes total sense!

File: _inc/users/template.php [hunk 1]
```diff
@@ -51,27 +53,30 @@ function bp_follow_get_add_follow_button( $args = '' ) {
...
+		if ( ! $r['leader_id'] || ! $r['follower_id'] ) {
 			return false;
+		}
+
+		$logged_user_id = bp_loggedin_user_id();
 
 		// if we're checking during a members loop, then follow status is already
 		// queried via bp_follow_inject_member_follow_status()
-		if ( ! empty( $members_template->in_the_loop ) && $r['follower_id'] == bp_loggedin_user_id() && $r['leader_id'] == bp_get_member_user_id() ) {
+		if ( ! empty( $members_template->in_the_loop ) && $r['follower_id'] === $logged_user_id && $r['leader_id'] === $logged_user_id ) {
```
[the remark this answers is not in the document]
Comment (renatonascalves) [reply _inc/users/template.php]: Fixed on 2f10e73.

**Comment by renatonascalves** [comment]
@r-a-y Do you need more updates or changes?

**Comment by r-a-y** [comment]
@renatonascalves - Appreciate the PR.  I'll take a look when I can and will report back.

**Comment by r-a-y** [comment]
I've merged this.  If there are issues, one of us should hopefully catch some bugs and fix them :)

Thanks @renatonascalves !

**Comment by renatonascalves** [comment]
Great! @r-a-y I have a site running with the WPCS changes without issues. But if something comes up, count on me to debug. :)

**Pull request merged**